privaterelay/templates/settings.html

```diff
@@ -0,0 +1,104 @@
+{% extends "base.html" %}
+
+{% load static %}
+{% load ftl %}
+{% ftlconf bundle='privaterelay.ftl_bundles.main' %}
+
+{% block javascript %}
+  <script src="{% static 'js/settings.js' %}"></script>
+{% endblock %}
+
+
+{% block content %}
+
+<main class="settings-main bg-light dashboard-container " data-api-token="{{ user_profile.api_token }}">
+  <div class="c-meta">
+    <h1 class="mzp-u-title-md c-settings-headline">{% ftlmsg 'settings-headline' %}</h1>
+    <ul class="c-meta--links">
+      <li>
+        <a
+          href="https://support.mozilla.org/en-US/kb/activate-firefox-relay-hide-your-email-address-alias?utm_source={{ settings.SITE_ORIGIN }}"
```

Comment:
We should remove the `en-US` part. SUMO has locale middleware that will automatically detect the browser language and redirect the user to the right translation. But if `en-US` is hard-coded into the url, this link will always only go to `en-US` translation of the page.

Reply:
Whoops good catch. As a non-American, I should've caught that really: d4ee27a